tests/unit/HelpContactUs.spec.ts

```diff
@@ -31,10 +31,10 @@ describe('HelpBusinessNumber', () => {
     expect(wrapper.find('.help-header h2').text()).toBe('Contact BC Registries')
 
     // verify paragraph
-    expect(wrapper.find('p').text()).toContain('If you require further assistance')
+    expect(wrapper.find('p').text()).toContain('If you require further assistance adding a business, please contact us.')
```

Comment:
Can you please make this to use two lines? We usually don't go over the length of 120 characters. I see GitHub complaining about that. 

If you run `npm run lint`, you can see all the warnings in regards to the maximum length. Please Shaoyun if you can fix it here and below.

Reply:
Fixed, thanks. Resolving.